handler.go

```diff
@@ -174,7 +216,7 @@ func (wac *Conn) dispatch(msg interface{}) {
 				for a := range con {
 					if v, ok := con[a].(*proto.WebMessageInfo); ok {
 						wac.handle(v)
-						wac.handle(parseProtoMessage(v))
+						wac.handle(ParseProtoMessage(v))
```

Comment:
@tulir It looks like to me that you changed to ParseProtoMessage instead of parseProtoMessage by accident or you forgot to also export the function in the message.go file

Reply:
Oh yeah, I needed that for something else and it's possible this branch accidentally includes other changes. That first commit (handle binary messages properly) is also not related to sync handlers.

Reply:
Splitting the fork into branches would be cleaner, but I don´t think is necessary here.